base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -154,7 +154,7 @@ impl ConsensusConstants {
             emission_initial: 5_538_846_115 * uT,
             emission_decay: 0.999_999_560_409_038_5,
             emission_tail: 1 * T,
-            min_pow_difficulty: 30_000.into(),
+            min_pow_difficulty: 30_000_000.into(),
```

Comment:
Just a note that this is not only the initial difficulty, but the minimum the difficulty can ever be

Reply:
This corresponds to <1 min in release mode on a single thread on a Macbook CPU, so it's still pretty low